src/olympia/versions/tests/test_models.py

```diff
@@ -920,6 +920,45 @@ def test_pending_rejection_property(self):
         flags.update(pending_rejection=in_the_past)
         assert version.pending_rejection == in_the_past
 
+    def test_pending_rejection_by_property(self):
+        addon = Addon.objects.get(id=3615)
+        version = addon.current_version
+        user = user_factory()
+        # No flags: None
+        assert version.pending_rejection_by is None
+        # Flag present, value is None (default): None.
+        flags = VersionReviewerFlags.objects.create(version=version)
+        assert flags.pending_rejection_by is None
+        assert version.pending_rejection_by is None
+        # Flag present, value is a user.
+        flags.update(pending_rejection_by=user)
+        assert version.pending_rejection_by == user
+
+    def test_pending_rejection_by_cleared_when_pending_rejection_cleared(self):
```

Comment:
I considered going through and adding assertions to every test where we are clearing `pending_rejection` (e.g., [`test_delete_clear_pending_rejection`](https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/tests/test_models.py#L624)), but I think this test, for the `save()` method, should suffice.